packages/icons/stories/icon.stories.tsx

```diff
@@ -87,3 +100,32 @@ export const AllIcons: ComponentStory<typeof Icon> = () => {
     </div>
   );
 };
+
+export const AllIconsWithSearch: ComponentStory<typeof Icon> = () => {
+  const [inputText, setInputText] = useState("");
+  return (
+    <StackLayout separators>
+      <FormField
+        label={"search icon"}
+        style={{ marginBlock: "1rem", maxWidth: "300px" }}
+      >
+        <Input value={inputText} onChange={(_, value) => setInputText(value)} />
+      </FormField>
+      <FlexLayout wrap gap={1}>
+        {allIconsNames
+          .filter(({ name, icon }) => new RegExp(inputText, "i").test(name))
+          .map(({ name, icon }, i) => {
+            return (
+              <StackLayout style={{ width: "150px" }} align="center">
+                {createElement(icon, {
+                  key: i,
+                  size: 2,
+                })}
+                <p>{name.slice(0, -1)}</p>
```

Comment:
What's the slice for? It seems to be cutting off the last letter of the name

Reply:
it was removing a comma, but it shouldn't be needed if we change the script